From caafb1986b2a8c5f714d228007960778898a7336 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Sun, 25 Nov 2018 13:54:33 +0800
Subject: [PATCH 9/9] caam_dma:fix a compile warning

Fix following compile warning:

WARNING: vmlinux.o(.data+0x8e618): Section mismatch in reference from the variable caam_dma_driver to the function .init.text:caam_dma_probe()
The variable caam_dma_driver references
the function __init caam_dma_probe()
If the reference is valid then annotate the
variable with __init* or __refdata (see linux/init.h) or name the variable:
*_template, *_timer, *_sht, *_ops, *_probe, *_probe_one, *_console

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/dma/caam_dma.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/dma/caam_dma.c b/drivers/dma/caam_dma.c
index e430b32..2e3b41c 100644
--- a/drivers/dma/caam_dma.c
+++ b/drivers/dma/caam_dma.c
@@ -442,7 +442,7 @@ static void set_caam_dma_desc(u32 *desc)
 #endif
 }
 
-static int __init caam_dma_probe(struct platform_device *pdev)
+static int caam_dma_probe(struct platform_device *pdev)
 {
 	struct device *dev = &pdev->dev;
 	struct device *ctrldev = dev->parent;
-- 
1.7.9.5

